ONSdigital/ras-frontstage
# Notify user to verify their email

# Motivation and Context
Currently, if an external user attempts to log in before they've verified their account, they are taken to a page and told that they must verify before continuing.  If they cannot find their original verification email from when they signed up then they have to contact us to request another verification email, as new verification emails can only be sent from response-operations.
This PR allows external users to request another verification email if they sign in with an unverified account.

# What has changed
- Amended existing template to allow external users to request another verification email
- Added a new template to inform users that the verification email has been sent
- Added functionality to send another verification email via the party service endpoint

# How to test?
- Create a new account but do not verify it
- Log in with the new account - you should be told that the account is locked and have the option to resend the verification email
- Click the link to resend the verification email.  A new mail should be sent (check the party service logs for the new verification URL)


# Links
Trello card --> https://trello.com/c/5LKDQyBI/282-us205-ext-notifying-a-user-to-verify-their-email-address-when-they-sign-in-to-an-unverified-account-m
